frontend/src/app/gitImport.tsx

```diff
@@ -0,0 +1,159 @@
+    const { branch, repository, isFormSubmitted, isMissingFields, isSucess } = this.state;
+    return (
+      <React.Fragment>
+        <Bullseye>
+          <div className="app-container">
+            <div>
+              {isMissingFields && (
+                <div className="notification-container">
+                  <Alert
+                    variant="warning"
+                    title="A repository url is required."
+                    action={<AlertActionCloseButton onClose={this.dismissNotification} />}
+                  />
+                </div>
+              )}
+              {isFormSubmitted && isSucess && (
+                <div className="notification-container">
+                  <Alert
+                    variant="success"
+                    title={this.state.submitMsg}
+                    action={<AlertActionCloseButton onClose={this.setRedirect} />}
+                  />
+                </div>
+              )}
+             {isFormSubmitted && !isSucess && (
+                <div className="notification-container">
+                  <Alert
+                    variant="danger"
+                    title={this.state.submitMsg}
+                    action={<AlertActionCloseButton onClose={this.dismissNotification} />}
+                  />
+                </div>
+              )}
+
+              <FormGroup
+                label="Repository URL"
+                fieldId="repository-url"
+              >
+                <TextInput id="repository-url" type="text" placeholder="Repository" value={repository} onChange={this.handleRepoInput} />
+              </FormGroup>
+              <br />
+              <FormGroup
+                label="Branch"
+                fieldId="branch-name"
+              >
+                <TextInput id="branch-name" type="text" placeholder="Branch" value={branch} onChange={this.handleBranchInput} />
+              </FormGroup>
+              <br />
+              <Button aria-label="Submit the repository and branch information to the git integration service." onClick={this.cloneRepo}>Submit</Button>
+              <div>
+                {this.loginRedirect()}
+                {this.renderRedirect()}
+              </div>
+            </div>
+          </div>
+        </Bullseye>
+      </React.Fragment>
+    );
+  }
+
+  private handleRepoInput = repository => {
+    this.setState({ repository });
+    console.log("Repository " + repository)
+  };
+
+  private handleBranchInput = branch => {
+    this.setState({ branch });
+    console.log("Branch " + branch)
+  };
+
+  private cloneRepo = (postBody) => {
+    console.log("My repo is: " + this.state.repository + " and branch is " + this.state.branch + ".")
+    if (this.state.repository === "") {
+      this.setState({ isMissingFields: true })
+    } else {
+      const hdrs = {
+        'Accept': 'application/json',
+        'cache-control': 'no-cache'
+      }
+
+      fetch('/content/repositories/GIT_INTEGRATION_URL')
```

Comment:
We should decide on a naming convention here, I'm not wild about the CAPS_CASE.

Reply:
renamed to pantGitServiceURL as we talked about.